<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaJogo</class>
 <widget class="QWidget" name="TelaJogo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: white;
}</string>
  </property>
  <widget class="QFrame" name="pino1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>8</width>
     <height>170</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>8</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>8</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color: rgb(122, 45, 0);
border-color: black;
border-width: 2 px;
border-right: 2 px;
border-left: 2 px;
border-style: outset;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="pino3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>8</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color: rgb(122, 45, 0);
border-color: black;
border-width: 2 px;
border-right: 2 px;
border-left: 2 px;
border-style: outset;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="pino2">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>40</y>
     <width>8</width>
     <height>170</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color: rgb(122, 45, 0);
border-color: black;
border-width: 2 px;
border-right: 2 px;
border-left: 2 px;
border-style: outset;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
